docs/sources/next/javascript-api/k6-experimental/browser/request/postdatabuffer.md

```diff
@@ -0,0 +1,51 @@
+---
+title: 'postDataBuffer()'
+description: 'Browser module: Request.postDataBuffer method'
+---
+
+# postDataBuffer()
+
+Request's post body in a binary form, if any.
+
+### Returns
+
+| Type                | Description                                                              |
+| ------------------- | ------------------------------------------------------------------------ |
+| ArrayBuffer \| null | Returns an ArrayBuffer with request's post data or null if no post data. |
```

Comment:
WDYT?

```suggestion
| ArrayBuffer \| null | Returns an [ArrayBuffer](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/ArrayBuffer) with request's post data or `null` if no post data. |
```

Reply:
I think ArrayBuffer is fine without the link, same as innerText. Having the link is one step further.

Reply:
OK.. Even so, we should put ArrayBuffer and null there in a code block.

Reply:
Agreed that ArrayBuffer and null should be in a code block. Changed in a3d428cad532235a816b8ab655c2d4a144a0089e

But what about references to promise? 😅

Reply:
What about them? 😊 Let’s assume some JS familarity from the reader.😅

Reply:
I mean, should they be placed in codeblocks too?

Reply:
And should it be `Promise` or `promise` 😅

Reply:
That’s a difficult question 😂 But to keep the noise low, I just say promise. We can find and replace later on if we decide otherwise.